### DISCUSSION TITLE
Question about DEVG options

### DISCUSSION THREAD

**Discussion by SmithSimmon** [NCAR/hrldas#164] [Q&A]
Hello, I have run the regional simulation successfully following the Note2 in tutorial.
After running the ./create_forcing.exe , I generate a series of forcing files below. I notice in the begining of each day, there are two more variables in the forcing file, which is "LAI" and "VEGFRA". 
![forcing_file](https://github.com/NCAR/hrldas/assets/162659864/2e71e2f0-3fc9-4bca-9d9d-0b4e34b75bee)
If I want to use time series of LAI satellite data, shoulde I add the "LAI" variable to my atmospheric forcing for each forcing files?(or just add "LAI" variable in the first forcing file of each day like the picture. e.g. 2010020200.LDASIN_DOMAIN1, 2010020300.LDASIN_DOMAIN1 ...)
by the way, I will choose the devg option to 9.

**Comment by cenlinhe** [comment]
you need to add LAI in each forcing file for each forcing timestep
